### PR TITLE
update WalletGeneration to 2.0

### PR DESCRIPTION
**Pull request by MarnixCroes**
will add/update img in follow up PR

### CODE REVIEW

File: docs/using-wasabi/WalletGeneration.md [hunk 1]
```diff
@@ -33,11 +36,9 @@ It encrypts your secrets, and you will need it every time you want to spend bitc
 	So, triple-check that you have a proper backup!
 	:::
 
-4. Click the `Generate` button.
-
-![Generate a Bitcoin wallet with Wasabi](/WalletManagerGenerateWallet.png "Generate a Bitcoin wallet with Wasabi")
+5. Confirm the password and click the `Continue` button.
 
-5. Write down the 12 recovery words (mnemonic seed phrase).
+6. Write down the 12 recovery words (mnemonic seed phrase).
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:41]:
```suggestion
6. Write down the 12 recovery words (mnemonic seed phrase) in the correct order and store them in a safe place.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 2]
```diff
@@ -33,11 +36,9 @@ It encrypts your secrets, and you will need it every time you want to spend bitc
...
 You can use this seed phrase **together with** your password to recover your wallet on a different computer using Wasabi or another [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliant wallet.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:42]:
```suggestion
You have to use this seed phrase **together with** your password to recover your wallet on a different computer using Wasabi or another [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliant wallet.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 3]
```diff
@@ -48,8 +49,16 @@ You can use this seed phrase **together with** your password to recover your wal
 
 ![Wasabi Wallet recovery words](/WalletManagerRecoveryWords.png "Wasabi Wallet recovery words")
 
-6. You must test the password before you can load the wallet, to make sure that your password is correct.
-So, type or paste the password in the text box, and click `Load Wallet`.
+7. You must confirm 3 of the 12 Recovery Words, as a quick check to make sure that your Recovery Words are correct.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:52]:
```suggestion
7. You must confirm 3 of the 12 Recovery Words, as a quick check to make sure that your Recovery Words are backed up.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 4]
```diff
@@ -18,13 +18,16 @@ So, with several wallets, you can conveniently manage your bitcoin for different
...
-2. Name the new wallet precisely to ensure a proper differentiation from wallets created in the future.
+2. Click `Create a new wallet` to generate a new wallet.
+
+3. Name the new wallet precisely to ensure a proper differentiation from wallets created in the future.
 This label is not shared with anyone, it is only stored locally on your computer.
+Click `Continue`.
 
-3. Write a long and random password and **[back it up](/using-wasabi/BackupBestPractices.md)**.
-It encrypts your secrets, and you will need it every time you want to spend bitcoin from this wallet or recover your wallet.
+4. Write a long and random password and **[back it up](/using-wasabi/BackupBestPractices.md)**.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:29]:
```suggestion
4. Write a long and random password (passphrase) and **[back it up](/using-wasabi/BackupBestPractices.md)**.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 5]
```diff
@@ -48,8 +49,16 @@ You can use this seed phrase **together with** your password to recover your wal
...
+7. You must confirm 3 of the 12 Recovery Words, as a quick check to make sure that your Recovery Words are correct.
+So, confirm the Recovery Words, and click `Continue`.
+
+8. Select the coinjoin strategy for this wallet and click `Continue`.
+
+9. The wallet is succesfully added!
+Click `Done`.
+
+10. The wallet is automatically selected.
+Enter the wallets' password to open the wallet.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:61]: IMO this step should be deleted because it has nothing to do with generating a wallet.

**Review by yahiheb — CHANGES_REQUESTED** [review]
Steps 6 and 7 should come before steps 4 and 5.

File: docs/using-wasabi/WalletGeneration.md (hunk 5 above)
Comment (MaxHillebrand) [inline docs/using-wasabi/WalletGeneration.md:61]:
```suggestion
```

**Comment by MarnixCroes** [comment]
> Steps 6 and 7 should come before steps 4 and 5.

ah yes that got changed recently, fixed it

**Review by yahiheb — APPROVED** [review]

**Pull request merged**